packages/flutter/lib/src/material/dialog.dart

```diff
@@ -86,6 +86,7 @@ class Dialog extends StatelessWidget {
   /// This sets the [Material.color] on this [Dialog]'s [Material].
   ///
   /// If `null`, [ColorScheme.surfaceContainerHigh] is used in Material 3.
+  /// Otherwise, defaults to [ThemeData.dialogBackgroundColor].
```

Comment:
We probably can also mention, in fullscreen dialog, the default background color is `ColorScheme.surface`.

Reply:
Great idea!